packages/terraform-modules-public/example/variables.tf

```diff
@@ -0,0 +1,211 @@
+# Provide the next vars with a vars-file or modifying the default value
+variable google {
+  description = "The GCP Data"
+  type        = map(string)
+
+  default = {
+    project = "my-project"
+    region  = "us-west1"
+    zone    = "us-west1-a"
+  }
+}
+
+variable network_name {
+  description = "The name of the new VPC network created"
+  type        = string
+
+  default = "celo-network"
+}
+
+variable celo_env {
+  description = "The celo network to connect with"
+  type        = string
+
+  default = "baklava"
+}
+
+variable network_id {
+  description = "The ethereum network ID"
+  type        = number
+  default     = 121119
+}
+
+variable ethstats_host {
+  description = "Ethstats host to report data"
+  type        = string
+  default     = "baklava-ethstats.celo-testnet.org"
+}
+
+variable geth_node_docker_image {
+  description = "The Celo Blockchain docker image"
+  type        = map(string)
+
+  default = {
+    repository = "us.gcr.io/celo-testnet/celo-node"
+    tag        = "baklava"
+  }
+}
+
+variable replicas {
+  description = "The replica number for each component"
+  type        = map(number)
+
+  default = {
+    validator           = 1 # Also used for proxy
+    txnode              = 0
+    attestation_service = 1
+  }
+}
+
+variable instance_types {
+  description = "The instance type for each component"
+  type        = map(string)
+
+  default = {
+    validator           = "n1-standard-1"
+    proxy               = "n1-standard-1"
+    txnode              = "n1-standard-1"
+    attestation_service = "n1-standard-1"
+  }
+}
+
+variable proxy_accounts {
+  description = "The account data for the proxy nodes"
+  type        = map
+
+  default = {
+    account_addresses = [
+      "0xF2...",
+    ]
```

Comment:
Right. I think I left there because it helps (at least me) for identifying the account I was using, although you can identify with any other parameter. I will update it with a comment. Thanks!